setup.cfg

```
@@ -74,7 +74,7 @@ docs =
     sphinxcontrib-httpdomain
     sphinxcontrib-svg2pdfconverter
 graph =
-    pygraphviz
+    pygraphviz<1.8
```

Hi. :wave: Coming here from `pygraphviz` Issue 395, and I'd like to suggest that you just exclude the broken version rather than capping `pygraphviz` below `1.8` artificially. 

```suggestion
    pygraphviz>1.0,!=1.8
```

`pygraphviz` has the correct `python_requires` (c.f. https://github.com/pygraphviz/pygraphviz/pull/394) so nothing will break in the future once they fix the `pip` `v22.X` issue they have now.

For more on this please see https://iscinumpy.dev/post/bound-version-constraints/

:+1: